### PR TITLE
fix: run dart fix

### PR DESCRIPTION
**Pull request by Tommytrg**

### CODE REVIEW

**Comment by parodyBit** [comment]
Happy to see this!
The repo has gone through several dart/flutter updates and updating the depreciated items clears some of the warning logs 🔥

**Review by parodyBit — APPROVED** [review]

**Comment by parodyBit** [comment]
the `name` property of an `enum` is only available since SDK 2.15.0-
so we should roll file `pubspec.yaml`
```
environment:
  sdk: ">=2.14.0 <3.0.0"
```
to 
```
  sdk: ">=2.15.0 <3.0.0"
```

**Review by gabaldon — CHANGES_REQUESTED** [review]
The color of the paddedButton in the dark mode is incorrect:

Incorrect:
Screenshot 2023-04-13 at 10 56 23
Correct:
Screenshot 2023-04-13 at 11 00 04

The commit where the ListView padding is deleted is missing:
Incorrect:
Screenshot 2023-04-13 at 10 56 32
Correct:
Screenshot 2023-04-13 at 10 59 49

**Review by gabaldon — COMMENTED** [review]
One last change:
Incorrect:
Screenshot 2023-04-13 at 16 11 18
Correct:
Screenshot 2023-04-13 at 16 15 26

File: lib/widgets/PaddedButton.dart [hunk 1]
```diff
@@ -30,10 +31,11 @@ class PaddedButton extends StatelessWidget {
     final theme = Theme.of(context);
 
     Widget primaryButton = ElevatedButton(
-      style: ElevatedButton.styleFrom(
+      style: theme.elevatedButtonTheme.style?.merge(ElevatedButton.styleFrom(
```
Comment (gabaldon) [inline lib/widgets/PaddedButton.dart:34]: instead add this to the light and dark theme
    `ElevatedButton.styleFrom(
        foregroundColor: Colors.white, // changes the color of the text`

**Review by gabaldon — CHANGES_REQUESTED** [review]

**Review by gabaldon — APPROVED** [review]

**Pull request merged**